### PR TITLE
Repermissioning journey page (wip)

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Makes the WIP reperm journey page more complete, adding steps and clickable fields. it's still very much a work in progress but as it's hidden behind a flag I'd rather merge this milestone in sooner than later

## Screenshots
screen shot 2017-11-27 at 11 03 40
screen shot 2017-11-27 at 11 03 51
screen shot 2017-11-27 at 11 03 56

### CODE REVIEW

**Review by mariogalic — APPROVED** [review]

File: identity/app/views/repermissionJourney.scala.html [hunk 1]
```diff
@@ -15,39 +18,85 @@
...
+            <div class="monocol-wrapper">
+
+                <div class="u-cf identity-section identity-wrapper">
+
+                    <div class="js-errorHolder manage-account__errors"></div>
+
+                    <div class="manage-account-wizard manage-account-wizard--repermission" >
+                        <div class="manage-account-wizard__step">
+                        <center>
+                            <img width="100%" src="https://i.imgur.com/TqI7Ub8.png" />
```
Comment (gustavpursche) [inline identity/app/views/repermissionJourney.scala.html]: `alt=""` is missing. Also: we shouldn't link an external image service shouldn't we?

File: identity/app/views/repermissionJourney.scala.html [hunk 2]
```diff
@@ -15,39 +18,85 @@
...
-            <div class="monocol-wrapper js-account-profile-forms">
+            <div class="monocol-wrapper">
+
+                <div class="u-cf identity-section identity-wrapper">
+
+                    <div class="js-errorHolder manage-account__errors"></div>
+
+                    <div class="manage-account-wizard manage-account-wizard--repermission" >
+                        <div class="manage-account-wizard__step">
+                        <center>
```
Comment (gustavpursche) [inline identity/app/views/repermissionJourney.scala.html]: `<center>` is deprecated: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center

File: identity/app/views/repermissionJourney.scala.html [hunk 3]
```diff
@@ -15,39 +18,85 @@
...
+                                                newsletter = newsletter
+                                            )(nonInputFields, messages, request, emailPrefsForm, emailSubscriptions)
+                                        </li>
+                                    }
+                                    </ul>
+                                </div>
+                            </form>
+                        </div>
+                        <div class="manage-account-wizard__step">
+                            <img class="js-manage-account-wizard__next" src="https://i.imgur.com/XoJmEro.png" width="100%" />
```
Comment (gustavpursche) [inline identity/app/views/repermissionJourney.scala.html]: Same here.

**Comment by walaura** [comment]
@gustavpursche these are all extremely dirty placeholders while we await for final art direction. I went with hotlinked images instead of code because they have been weirdly useful at giving stakeholders a feel for the final flow!

**Comment by gustavpursche** [comment]
@walaura I'd rather leave them empty in this case, instead of deploying this. Maybe upload an image to a guardian S3 bucket and use this one, but I'm definitely 👎 on the external placeholder.

**Comment by walaura** [comment]
@gustavpursche i get your point gonna remove them for now as that's faster

**Review by gu-stav — APPROVED** [review]
Thanks!

**Pull request merged**